fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/domain/LoanProduct.java

```diff
@@ -232,6 +232,9 @@ public class LoanProduct extends AbstractPersistableCustom<Long> {
     @Column(name = "repayment_start_date_type_enum", nullable = false)
     private RepaymentStartDateType repaymentStartDateType;
 
+    @Column(name = "enable_accrual_activity_posting", nullable = false)
+    private boolean enableAccrualActivityPosting = false;
```

Comment:
Same as with Loan entity. `LoanProductRelatedDetail` would be a better place for this